logstash/datadog_checks/logstash/logstash.py

```diff
@@ -172,37 +180,46 @@ def _get_logstash_version(self, config):
         self.log.debug("Logstash version is %s", version)
         return version
 
+    def _is_multi_pipeline(self, version):
+        """ Reusable version checker """
+        if version and LooseVersion(version) < LooseVersion("6.0.0"):
+            return False
+        return True
```

Comment:
```suggestion
        return version and LooseVersion(version) >= LooseVersion("6.0.0"):
```
Nit: In the case where the version is not available, we should return False